tests/modules/nf-core/ascat/main.nf

```diff
@@ -1,114 +0,0 @@
-#!/usr/bin/env nextflow
-
-nextflow.enable.dsl = 2
-
-include { ASCAT as ASCAT_SIMPLE }            from '../../../../modules/nf-core/ascat/main.nf'
-include { ASCAT as ASCAT_PLOIDY_AND_PURITY } from '../../../../modules/nf-core/ascat/main.nf'
-include { ASCAT as ASCAT_CRAM }              from '../../../../modules/nf-core/ascat/main.nf'
-include { UNZIP as UNZIP_ALLELES }           from '../../../../modules/nf-core/unzip/main.nf'
-include { UNZIP as UNZIP_LOCI }              from '../../../../modules/nf-core/unzip/main.nf'
-include { UNZIP as UNZIP_GC }                from '../../../../modules/nf-core/unzip/main.nf'
-include { UNZIP as UNZIP_RT }                from '../../../../modules/nf-core/unzip/main.nf'
-
-workflow test_ascat {
-    input = [
-        [ id:'test', single_end:false ], // meta map
-        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
-        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
-        file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam'], checkIfExists: true),
-        file(params.test_data['homo_sapiens']['illumina']['test2_paired_end_sorted_bam_bai'], checkIfExists: true)
-    ]
-
-    ASCAT_SIMPLE ( input , [], [], [], [], [], [])
-}
-
-// extended tests running with 1000 genomes data. Data is downloaded as follows:
-// wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/HG00154/alignment/HG00154.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam
-// wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/HG00154/alignment/HG00154.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam.bai
-// wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/HG00155/alignment/HG00155.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam
-// wget http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/HG00155/alignment/HG00155.mapped.ILLUMINA.bwa.GBR.low_coverage.20101123.bam.bai
-// wget https://www.dropbox.com/s/l3m0yvyca86lpwb/G1000_loci_hg19.zip
-// wget https://www.dropbox.com/s/3fzvir3uqe3073d/G1000_alleles_hg19.zip
-// wget https://www.dropbox.com/s/v0tgr1esyoh1krw/GC_G1000_hg19.zip
-// wget https://www.dropbox.com/s/50n7xb06x318tgl/RT_G1000_hg19.zip
-
```

Comment:
It would be nice if we could keep this info somewhere for local testing on input file changes. maybe we can add a readme here

Reply:
Good point! I'm going to save it in a readme; where do you suggest I should save it? In _modules/nf-core/ascat/tests_?

Reply:
I think that is a good palce